cvrf2csaf/cvrf2csaf.py

```diff
@@ -180,9 +207,14 @@ def main():
     h = DocumentHandler(config)
     final_csaf = h.convert_file(path=config.get('input_file'))
 
-    if not h.validate_output_against_schema(config.get('output_file')):
-        # TODO: If --force given, the output should not be written when CSAF not valid according to schema
-        pass
+    if not h.validate_output_against_schema(config.get('output_file')) or SectionHandler.error_occurred:
```

Comment:
How does that work? `h.validate_output_against_schema(config.get('output_file')) ` is intended to use the output file which is written in line 221. Shouldn't we pass `final_csaf` here instead and not read the document from file?

Reply:
This is already changed in #54 so I left it here untouched.